boa/benches/exec.rs

```diff
@@ -11,24 +11,29 @@ use criterion::{black_box, criterion_group, criterion_main, Criterion};
 static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
 
 static SYMBOL_CREATION: &str = r#"
-let a = Symbol();
-let b = Symbol();
-let c = Symbol();
+var a = Symbol();
+var b = Symbol();
+var c = Symbol();
 "#;
 
 fn create_realm(c: &mut Criterion) {
     c.bench_function("Create Realm", move |b| b.iter(Realm::create));
 }
 
 fn symbol_creation(c: &mut Criterion) {
+    // Create new Realm
+    let realm = Realm::create();
+    let mut engine = Interpreter::new(realm);
+    forward(&mut engine, SYMBOL_CREATION); // In order to initialize variables
+
     c.bench_function("Symbols (Execution)", move |b| {
-        b.iter(|| exec(black_box(SYMBOL_CREATION)))
+        b.iter(|| forward(&mut engine, black_box(SYMBOL_CREATION)))
```

Comment:
The `forward` function also lexes and parser, which would mess with the result benchmarks

Reply:
`forward` should be more accurate as its not creating a new interprerter and realm all over again, but yes it will still lex and parse the new input, i don't see how you can avoid that.

Reply:
I think we can avoid this by lexing and parsing before. Let me see if I can improve it.